### ISSUE TITLE
*.md installed in top-level site-packages, not in site-packages/requests_cache-0.7.3.dist-info/

### ISSUE THREAD

**Issue by denis-bz** [reclosedev/requests-cache#351]
### The problem
_A description of what the bug is, including a complete traceback (if applicable)_
```
pip install requests_cache
pip uninstall requests_cache  # to list, or else requests_cache-0.7.3.dist-info/RECORD
  Would remove:
    /opt/local/py3/lib/python3.7/site-packages/.readthedocs.yml    top level ❓
    /opt/local/py3/lib/python3.7/site-packages/CODESHELTER.md
    /opt/local/py3/lib/python3.7/site-packages/CONTRIBUTING.md
    /opt/local/py3/lib/python3.7/site-packages/CONTRIBUTORS.md
    /opt/local/py3/lib/python3.7/site-packages/HISTORY.md
    /opt/local/py3/lib/python3.7/site-packages/README.md
    /opt/local/py3/lib/python3.7/site-packages/docker-compose.yml
    /opt/local/py3/lib/python3.7/site-packages/requests_cache-0.7.3.dist-info/*
    /opt/local/py3/lib/python3.7/site-packages/requests_cache/*
```
### Expected behavior
_A description of what you expected to happen_
*.md in .../site-packages/requests_cache-0.7.3.dist-info

### Steps to reproduce the behavior
_With a complete code example, if possible_
`pip install --prefix a-new-directory requests_cache`
or make a wheel, list it ? what files go into a dist / a wheel is murky to me

### Workarounds
_Is there an existing workaround for this issue?_

### Environment
- requests-cache version: 0.7.3
- Python version: 3.7.10
- Platform: macos 10.10
- pip 21.2.3  wheel 0.37.0

**Comment by JWCook** [comment]
Interesting, I wasn't aware of that behavior. Thanks for the heads up!

I switched the packaging config to Poetry for 0.7, and this must be a case of [`tool.poetry.include`](https://github.com/reclosedev/requests-cache/blob/v0.7.3/pyproject.toml#L11-L17) behaving differently from the previous [MANIFEST.in](https://github.com/reclosedev/requests-cache/blob/v0.6.4/MANIFEST.in). I'll look into it some more.

**Comment by denis-bz** [comment]
@JWCook, could you ping the poetry guys
and maybe wheel-maker guys (dunno who) to flag wheels with toplevel files,
to see that sooner ?

Re MANIFEST.in: if it works, don't fix it :)

**Comment by JWCook** [comment]
I don't believe this is an issue with poetry or wheels; I think those extra files just aren't supposed to be included in wheels in general (even in `*.dist-info/`). 

I updated the config to include `*.md` in the sdist only, and there will probably be a `0.7.4` patch release within a week or so.

**Comment by JWCook** [comment]
@denis-bz This is fixed in requests-cache `0.7.4`.
